src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -325,9 +310,9 @@ private BindingConfig GetDefaultBindingConfig()
         /// Optional hook to customize intellisense. 
         /// </summary>
         /// <returns></returns>
-        private protected virtual IIntellisense CreateIntellisense()
+        private protected virtual IIntellisense CreateIntellisense(CultureInfo culture)
```

Comment:
>culture [](http://example.com/codeflow?start=79&length=7)

I wonder if we should start using more specific terms, like "parserCulture"

Reply:
I can't be more in agreement
Too many parameters or function names aren't precise enough or just wrongly chosen (history... I know)